addons/resource.language.en_gb/resources/strings.po

```diff
@@ -900,11 +900,13 @@ msgctxt "#203"
 msgid "Plot outline"
 msgstr ""
 
+#. label of a smartplaylist filter option to filter for albums on compilations
 #: xbmc/playlists/SmartPlaylist.cpp
 msgctxt "#204"
 msgid "Compilation"
 msgstr ""
 
+#. label of a smartplaylist filter option to filter for songs or albums on votes for rating
```

Comment:
When importing an album rating from an online source you can also get a count of how many people rated the album, that is "votes". I think, as I didn't add and don't use, this bit of music functionality.